### ISSUE TITLE
Reproduce of training

### ISSUE THREAD

**Issue by JinyuanSun** [nekitmm/DLPacker#3]
Hi, 
I am trying to train dlpacker from scratch. How did the learning rate been adjusted? In the paper, you mentioned a decrease when the loss is not improving, I am not sure there is a more strict definition.

**Comment by nekitmm** [comment]
I think Figure 1 in the paper illustrates this pretty well. Not sure what I can add here to answer your question!
